kfp/kfp_support_lib/src/kfp_support/workflow_support/utils/kfp_utils.py

```diff
@@ -0,0 +1,109 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import json
+import os
+import re
+import sys
+from typing import Any
+
+from data_processing.utils import get_logger
+
+
+logger = get_logger(__name__)
+
+ONE_HOUR_SEC = 60 * 60
```

Comment:
do we want it in each file? They are defined in `component_utils.py` as well

Reply:
No, cut and paste. Fixed